src/main/java/apoc/bolt/Bolt.java

```diff
@@ -0,0 +1,160 @@
+package apoc.bolt;
+
+import apoc.Description;
+import apoc.result.RowResult;
+import apoc.result.VirtualNode;
+import apoc.result.VirtualRelationship;
+import apoc.util.UriResolver;
+import apoc.util.Util;
+import org.neo4j.driver.internal.InternalEntity;
+import org.neo4j.driver.internal.InternalPath;
+import org.neo4j.driver.v1.*;
+import org.neo4j.driver.v1.summary.SummaryCounters;
+import org.neo4j.driver.v1.types.Node;
+import org.neo4j.driver.v1.types.Path;
+import org.neo4j.driver.v1.types.Relationship;
+import org.neo4j.graphdb.GraphDatabaseService;
+import org.neo4j.graphdb.Label;
+import org.neo4j.graphdb.RelationshipType;
+import org.neo4j.procedure.Context;
+import org.neo4j.procedure.Name;
+import org.neo4j.procedure.Procedure;
+
+import java.net.URISyntaxException;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.stream.Stream;
+
+import static apoc.util.MapUtil.map;
+
+/**
+ * @author AgileLARUS
+ * @since 29.08.17
+ */
+public class Bolt {
+
+    @Context
+    public GraphDatabaseService db;
+
+    @Procedure()
+    @Description("apoc.bolt.load(url-or-key, statement, params, config) - access to other databases via bolt for read")
+    public Stream<RowResult> load(@Name("url") String url, @Name("statement") String statement, @Name(value = "params", defaultValue = "{}") Map<String, Object> params, @Name(value = "config", defaultValue = "{}") Map<String, Object> config) throws URISyntaxException {
+
+        if (params == null) params = Collections.emptyMap();
+        if (config == null) config = Collections.emptyMap();
+        boolean virtual = (boolean) config.getOrDefault("virtual", false);
+        boolean addStatistics = (boolean) config.getOrDefault("statistics", false);
+        UriResolver uri = new UriResolver(url, "bolt");
+        uri.initialize();
+        Driver driver = null;
+        Session session = null;
+        try {
+            driver = GraphDatabase.driver(uri.getUrlDriver(), AuthTokens.basic(uri.getUser(), uri.getPassword()));
```

Comment:
allow "driverConfig" be passed to the driver configuration: 
e.g. logging, tls, ... via a `Config toDriverConfig(config.getOrDefault("driverConfig",map())`
http://neo4j.com/docs/api/java-driver/current/org/neo4j/driver/v1/GraphDatabase.html#driver-java.net.URI-org.neo4j.driver.v1.AuthToken-org.neo4j.driver.v1.Config-